gui/util/range_slider.py

```diff
@@ -216,39 +234,56 @@ def paintEvent(self, event):
         painter.drawRect(self.display_max, 1, self.bar_width, h - 2)  # right
 
     def rangeSliderSize(self):
-        """
-        :return: width attribute of parent class QWidget
+        """Size of the slider.
+
+        Returns
+        -------
+        size : int
+            Slider widget width.
```

Comment:
slider could be vertical too :D saying only width could be deceptive.

Reply:
That's why the method summary is neutral in language. However, I thought that we should specify that it is the width in this case.

Reply:
Eh, I'm with @AhmetCanSolak, I think width only here could be confusing. We don't have to be so concise here, let's clarify: 

```suggestion
            Slider widget width (horizontal sliders) or height (vertical sliders).
```